webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/repository/jdbc/ReportingServiceTest.scala

```diff
@@ -101,34 +106,20 @@ class ReportingServiceTest extends DBCommon with BoxSpecMatcher {
     transacRun(xa => sql"DELETE FROM ReportsExecution; DELETE FROM RudderSysEvents;".update.run.transact(xa))
   }
 
-  object nodeInfoService extends NodeInfoService {
-    def getNodeInfo(nodeId: NodeId):           IOResult[Option[NodeInfo]]      = ???
-    def getNodeInfoPure(nodeId: NodeId):       IOResult[Option[NodeInfo]]      = ???
-    def getNodeInfos(nodesId: Set[NodeId]):    IOResult[Set[NodeInfo]]         = ???
-    def getNodeInfosSeq(nodeIds: Seq[NodeId]): IOResult[Seq[NodeInfo]]         = ???
-    def getNode(nodeId: NodeId):               Box[Node]                       = ???
-    def getAllNodes():                         IOResult[Map[NodeId, Node]]     = ???
-    def getAllNodesIds():                      IOResult[Set[NodeId]]           = ???
-    def getAllNodeInfos():                     IOResult[Seq[NodeInfo]]         = ???
-    def getAllSystemNodeIds():                 IOResult[Seq[NodeId]]           = ???
-    def getPendingNodeInfos():                 IOResult[Map[NodeId, NodeInfo]] = ???
-    def getPendingNodeInfo(nodeId: NodeId):    IOResult[Option[NodeInfo]]      = ???
-    def getDeletedNodeInfos():                 IOResult[Map[NodeId, NodeInfo]] = ???
-    def getDeletedNodeInfo(nodeId: NodeId):    IOResult[Option[NodeInfo]]      = ???
-    def getNumberOfManagedNodes:               IOResult[Int]                   = ???
-    val getAll:                                IOResult[Map[NodeId, NodeInfo]] = {
-      def build(id: String, mode: Option[PolicyMode]) = {
-        val node1 = NodeConfigData.node1.node
-        NodeConfigData.node1.copy(node = node1.copy(id = NodeId(id), policyMode = mode))
-      }
-      Seq(
-        build("n0", None),
-        build("n1", Some(PolicyMode.Enforce)),
-        build("n2", Some(PolicyMode.Audit)),
-        build("n3", Some(PolicyMode.Enforce)),
-        build("n4", Some(PolicyMode.Audit))
-      ).map(n => (n.id, n)).toMap.succeed
+  val nodeFactRepo = {
+    def build(id: String, mode: Option[PolicyMode]) = {
+      (NodeId(id), NodeConfigData.fact1.modify(_.id).setTo(NodeId(id)).modify(_.rudderSettings.policyMode).setTo(mode))
     }
+
+    val accepted = Seq(
+      build("n0", None),
+      build("n1", Some(PolicyMode.Enforce)),
+      build("n2", Some(PolicyMode.Audit)),
+      build("n3", Some(PolicyMode.Enforce)),
+      build("n4", Some(PolicyMode.Audit))
+    ).toMap
+
+    CoreNodeFactRepository.make(NoopFactStorage, NoopGetNodesbySofwareName, Map(), accepted, Chunk.empty).runNow
```

Comment:
The base repo for `ReportingServiceTest` is now a fact repo. It doesn't really matters since all tests are commented out in any case.